react/Header/UserAccountMenu/UserAccountMenu.js

```diff
@@ -57,31 +62,35 @@ export default function UserAccountMenu({ locale, authenticationStatus, linkRend
           })
         }
       </li>
-      <li className={classnames(styles.aboveSmallDevice, activeTab === 'Saved Searches' && styles.activeTab)}>
-        {
-          linkRenderer({
-            'data-analytics': 'header:saved+searches',
-            className: `${styles.item} ${styles.subItem}`,
-            href: '/myactivity#favourite',
-            children: [
-              <span key="label">Saved Searches</span>,
-              <HeartIcon
-                key="icon"
-                className={classnames(styles.icon, styles.saveSearches)}
-                svgClassName={styles.iconSvg}
-              />
-            ]
-          })
-        }
-      </li>
+
+      <Hidden mobile>
```

Comment:
Sorry for this pick, that was really random, I'm worried that putting ul li inside span doesn't work in all browsers (and it's invalid)...I'll play around there probably. That was bugging my brain long enough, I guess we need Hidden to accept a component